### PR TITLE
Upgrade Brave Alert Lib (CU-860phzbq5)

### PR DESCRIPTION
**Pull request by trisapeace**
- Which has the latest version of Twilio node, so was able to remove the audit-ci exception

## Test plan
* [x] Deploy to dev
* [x] Run smoke tests
   * [x] Text messages are sent
   * [x] Text messages are received

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-860phzbq5 Upgrade Twilio node in Buttons, Sensors, BraveAlertLib, and PA](https://app.clickup.com/t/860phzbq5)

File: server/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "@sentry/node": "^6.2.5",
     "@sentry/tracing": "^6.2.5",
     "axios": "^0.21.2",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v9.1.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-upgrade-twilio",
```
Comment (trisapeace) [inline server/package.json:22]: Remember to update this once https://github.com/bravetechnologycoop/brave-alert-lib/pull/54 has been merged

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,11 +14,8 @@ the code was deployed.
 ### Changed
 
 - Updated to Device OS 3.3.1 (CU-3aru0mb).
-<<<<<<< HEAD
 - Replaced locationid with client display name in Sensor disconnection initial and reminder messages (CU-2q39wpk).
-=======
 - Sensor heartbeat API call body key changed from 'doorLastHeartbeat' to 'doorLastMessage' (CU-34atvnc).
->>>>>>> 1e43212 (Change the door last seen behaviour)
```
Comment (trisapeace) [inline CHANGELOG.md:21]: Oops, somehow these got in during the last merge conflict. We'll fix that here

Comment (susanchung-brave) [reply CHANGELOG.md:21]: oop thanks for the catch!

Comment (trisapeace) [reply CHANGELOG.md]: Haha, yup! Funny.

File: server/package.json (hunk 1 above)
Comment (trisapeace) [reply server/package.json]: Done.

**Pull request merged**